test/features/operator/install_explainability.feature

```diff
@@ -6,17 +6,71 @@ Feature: Kogito Explainability service
     Given Namespace is created
 
   @smoke
-  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
-  @disabled
-  Scenario: Install Kogito Explainability
+  Scenario Outline: Install Kogito Explainability with communication <communication>
     Given Kogito Operator is deployed with Kafka operator
-    And Install Kogito Explainability with 1 replicas
+    And Kafka topic "trusty-explainability-result" is deployed
+    When Install Kogito Explainability with 1 replicas with configuration:
+      | runtime-env  | EXPLAINABILITY_COMMUNICATION   | <communication>   |
     Then Kogito Explainability has 1 pods running within 10 minutes
-    And HTTP GET request on service "explainability" is successful within 2 minutes with path "health/live"
+
+    Examples:
+      | communication | 
+      | messaging     | 
+      | rest          | 
+
+#####
+
+  @events
+  @infinispan
+  @kafka
+  Scenario Outline: Explainability retrieves explainability requests events with communication <communication>
+    Given Kogito Operator is deployed with Infinispan and Kafka operators
+    And Install Kogito Trusty with 1 replicas
+
+    When Install Kogito Explainability with 1 replicas with configuration:
+      | runtime-env  | EXPLAINABILITY_COMMUNICATION   | <communication>   |
+    And Clone Kogito examples into local directory
+    And Local example service "dmn-tracing-quarkus" is built by Maven using profile "default" and deployed to runtime registry
+    And Deploy quarkus example service "dmn-tracing-quarkus" from runtime registry with configuration:
+      | config     | enablePersistence | enabled |
+      | config     | enableEvents      | enabled |
+    And Kogito Runtime "dmn-tracing-quarkus" has 1 pods running within 10 minutes
+    And HTTP POST request on service "dmn-tracing-quarkus" is successful within 2 minutes with path "LoanEligibility" and body:
+      """json
+      {
+      "Bribe": 100,
+      "Client": {
+        "age": 45,
+        "existing payments": 2000,
+        "salary": 2000
+      },
+      "Loan": {
+        "duration": 40,
+        "installment": 1000
+      },
+      "SupremeDirector": "yes"
+      }
+      """
+
+    Then Explainability result for execution "LoanEligibility" in the Trusty service within 3 minutes with saliences:
+      | Eligibility  | 
+      | Is Enought?  |
+
+    Examples:
+      | communication | 
+      | messaging     | 
+
+    # Disabled due to https://issues.redhat.com/browse/KOGITO-3373: Trusty Service errors when running Explainability service with rest communication
+    @disabled
+    Examples:
+      | communication | 
+      | rest          | 
```

Comment:
Hi @r00ta , I made this test but it's not working due to some errors in the Trusty service (I'm getting 400 http error when getting the decisions data). I've raised KOGITO-3373 to solve this, but let me know I'm doing something wrong.

Reply:
I've removed this scenario following your comment in the ticket. We can continue clarifying in the ticket directly.